pola/management/commands/import_brands.py

```diff
@@ -0,0 +1,86 @@
+import csv
+import re
+
+from django.core.management import BaseCommand
+
+from company.models import Brand, Company
+from product.models import Product
+
+
+def update_product(self, brand, ean_code, company, product_name):
+    if Product.objects.filter(code=ean_code).count():
+        product = Product.objects.get(code=ean_code)
```

Comment:
```suggestion
    product = Product.objects.filter(code=ean_code).first()
    if productt:
```
To jest troche wydajniej oraz czytelniej, bo mamy mniej kodu.